bin/htmlproofer

```diff
@@ -71,6 +71,8 @@ Mercenary.program(:htmlproofer) do |p|
 
     options[:cache] = {}
     options[:cache][:timeframe] = opts['timeframe'] unless opts['timeframe'].nil?
+    
+    options[:http_status_ignore] = Array(options[:http_status_ignore]).map(&:to_i)
```

Comment:
Can you add a test for it? In `irb` at least I get:

```
Array("421, 422").map(&:to_i)
=> [421]
irb(main):005:0> 
```

https://github.com/gjtorikian/html-proofer/blob/4434b7eecb19d869c59c92e6480661d18866253c/spec/html-proofer/command_spec.rb is where all the `bin` tests go. :heart: